webpack.config.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path')
```

Comment:
Seperate the webpack config for dev and production environment. 

You can have a base config that is extended by both the dev and prod config in a folder in the root directory.

Reply:
ok